### ISSUE TITLE
Return to existing script line when returning to Manager script

### ISSUE THREAD

**Issue by her123** [APSIMInitiative/ApsimX#2440]
When selecting between Manager components it would be nice to return to the last position in the script for each manager as APSIM Classic does.

**Comment by hol430** [comment]
@zur003 Did you mean to resolve this issue via #2524?

**Comment by zur003** [comment]
Yes, I did mean to resolve it. It looks like I put in the number of a previous pull request rather than the issue when indicating what was resolved.
